### PR TITLE
Add URL field to card chooser

### PR DESCRIPTION
**Pull request by habdelra**

### CODE REVIEW

**Review by burieberry — APPROVED** [review]

File: packages/host/app/services/operator-mode-state-service.ts [hunk 1]
```diff
@@ -171,7 +171,9 @@ export default class OperatorModeStateService extends Service {
     if (this.recentCards.length > 10) {
       this.recentCards.splice(0, 1);
     }
-    const recentCardIds = this.recentCards.map((recentCard) => recentCard.id);
+    const recentCardIds = this.recentCards
+      .map((recentCard) => recentCard.id)
+      .filter(Boolean); // don't include cards that don't have an ID
```
Comment (jurgenwerk) [inline packages/host/app/services/operator-mode-state-service.ts:176]: Nice catch

**Review by jurgenwerk — APPROVED** [review]
Looking good

**Review by tintinthong — APPROVED** [review]
LGTM

**Comment by habdelra** [comment]
@burieberry the picker and URL field are now mutually exclusive

**Pull request merged**